test/parser/parser_test.cpp

```diff
@@ -99,6 +99,42 @@ TEST_F(ParserTestBase, CreateFunctionTest) {
       "BEGIN RETURN i + 1; END; $$ "
       "LANGUAGE plpgsql;";
   stmts = pgparser.BuildParseTree(query);
+
+  query =
```

Comment:
We probably don't want to have too many FUNCTION tests since it's going to be a while until we support pl/pgsql again.